src/main/java/com/jfb/tickets/config/TicketsContextConfiguration.java

```diff
@@ -0,0 +1,39 @@
+package com.jfb.tickets.config;
+
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.context.annotation.Scope;
+
+import com.jfb.tickets.dao.TicketDAO;
+import com.jfb.tickets.dao.UserDAO;
+import com.jfb.tickets.service.TicketService;
+import com.jfb.tickets.service.UserService;
+
+@Configuration
+public class TicketsContextConfiguration {
+    @Bean
+    @Scope("singleton")
```

Comment:
by default all beans are singletons, here you can remove it. specify bean scope with `@Scope` only when you need some different scope.